### PR TITLE
Test that a mixin getter is not treated as a concrete field.

### PR DESCRIPTION
**Pull request by jifalops**
Sorry if this should go to a different branch. This is just a test to help support #37810

### CODE REVIEW

**Comment by jifalops** [comment]
Sorry if this should go to a different branch. This is just a test to help support https://github.com/dart-lang/sdk/issues/37810

**Comment by mit-mit** [comment]
Hi @jifalops, we're not able to process this PR unless the CLA is signed.

**Comment by jifalops** [comment]
I signed it two or three days ago, not sure why it's not showing up.

**Comment by athomas** [comment]
For posterity: once signing the CLA, googlebot needs to be notified by adding a comment like this (see the text of the googlebot comment):

`@googlebot I signed it!`

File: pkg/analyzer/test/src/diagnostics/const_constructor_with_mixin_with_field_test.dart [hunk 1]
```diff
@@ -105,6 +105,18 @@ class X extends Object with M {
     await assertNoErrorsInCode('''
 mixin M {}
 
+class X extends Object with M {
+  const X();
+}
+''');
+  }
+
```
Comment (athomas) [inline pkg/analyzer/test/src/diagnostics/const_constructor_with_mixin_with_field_test.dart]: Please remove the trailing whitespace on this line.

**Comment by mit-mit** [comment]
Code review https://dart-review.googlesource.com/c/sdk/+/116262

**Comment by mit-mit** [comment]
Closing as the feedback from code review wasn't incorporated

**Pull request closed without merging**